jenkinsci/jenkins-automation-operator
# Sync with upstream Jan 22 2021 (#9)

* Add checks and cleanup to Test Suite

* Add github workflow

* Fix github actions yaml syntax

* Add go github workflow

* Add persistentSpec to Jenkins

* Remove e2e.yml in favour of go.yml for github actions

* move to new upstream repo jenkins-automation-operator

* add prometheus service monitor for jenkins

* check for prometheus api available

* fix permissions jenkinsimages finalizers

* Fix linting after syncing with downstream

Co-authored-by: jkhelil <jkhelil@redhat.com>
